### PR TITLE
[New Sample] - Tab-Meeting-Auto-Recording - Csharp

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |25| Tab External Auth | This sample illustrates how to implement Google auth using external auth providers. |[View][tab-external-auth#csharp]||
 |26| Tab Meeting Tanscript Recording |This sample shows meeting transcripts and recordings. |[View][tab-meeting-transcript-recording#csharp]|[View][tab-meeting-transcript-recording#nodejs] ![toolkit-icon](assets/toolkit-icon.png)|
 |27| Tab DeepLink Pass Values |Microsoft Teams sample which displays deeplinks based on the values passed dynamically to the context. ||[View][tab-deeplink-pass-values#ts]|
-
+|28| Tab Meeting Auto Recording |This sample shows meeting auto recording. |[View][tab-meeting-auto-recording#csharp]||
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply README.md:79]: Fixed

File: samples/tab-meeting-auto-recording/csharp/README.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+extensions:
+ contentType: samples
+ createdDate: "30-07-2024 10:00:01"
+urlFragment: officedev-microsoft-teams-samples-tab-meeting-auto-recording-csharp
+
+---
+
+## Teams Tab - Meeting Auto-Recording
+
+This feature ensures that meetings initiated from a specific Teams tab are automatically recorded, leveraging event subscriptions to manage and trigger the recording process. The implementation involves subscribing to relevant Teams events and integrating with the Microsoft Graph API to control recording settings.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-meeting-auto-recording/csharp/README.md:19]: Fixed

File: samples/tab-meeting-auto-recording/csharp/README.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.10 Preview 4  or higher** [Visual Studio](https://visualstudio.microsoft.com/downloads/)
+1. Install Teams Toolkit for Visual Studio [Teams Toolkit extension](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+1. In the debug dropdown menu of Visual Studio, select default startup project > **Microsoft Teams (browser)**
+1. In Visual Studio, right-click your **TeamsApp** project and **Select Teams Toolkit > Prepare Teams App Dependencies**
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps.
+1. Select **Debug > Start Debugging** or **F5** to run the menu in Visual Studio.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-meeting-auto-recording/csharp/README.md:102]: Updated

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/10179066735

image

**Review by devjha-ms — APPROVED** [review]
Looks good. Thanks!

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline -

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/10244159573

image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Pull request merged**